### DISCUSSION TITLE
new notif highlighting

### DISCUSSION THREAD

**Discussion by louwonder** [AprilSylph/XKit-Rewritten#1165] [Suggestion Box]
just today my notifications have started highlighting the top one in DARk dark blue (rendering it illegible since the type on it is still black)- it's bothering me more than seems at all reasonable, I really can't stand it. Hoping you can roll out a tweak to make it stop??

**Comment by marcustyphoon** [comment]
Got a screenshot?

**Reply by louwonder** [reply]
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/140465382/4480792f-ca70-4c2f-8229-39f49746688c)
light blue highlighting for all unseen, used to that now, but now just the very top one always in this @*$&@ dark blue

**Reply by louwonder** [reply]
oh! it goes light (white background) if I hover on it. but I just want it to not do it

**Reply by marcustyphoon** [reply]
Glad I asked for a screenshot! Send that in to tumblr.com/support; I can't imagine that isn't a bug.

**Reply by louwonder** [reply]
I put nothing past them as an "innovation" but I'll send it along, lets hope you're right

**Reply by louwonder** [reply]
a completely separate account of mine (different login), in a different browser is also doing it... I haven't seen anyone else mention it but how could that happen if it was a bug? I did submit to support but I'm not exactly holding my breath on getting a reply

**Reply by marcustyphoon** [reply]
Do you have any extensions that are installed in both browsers?

**Reply by louwonder** [reply]
only adblocks in the secondary browser, and they are not the same ones as I use in my primary

**Reply by louwonder** [reply]
omg it's evolving 
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/140465382/32cb66b6-25b9-4a28-a885-b31a6e1e2823)

**Reply by louwonder** [reply]
oooh wait I was wrong- I have old xkit on both. But I tried disabling manually every separate xkit extension in one (couldn't see a way to paus the whole thing) and it still does it
